src/main/java/salesforce/ui/pages/lightning/campaign/CampaignPage.java

```diff
@@ -0,0 +1,30 @@
+package salesforce.ui.pages.lightning.campaign;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import salesforce.ui.pages.BasePage;
+
+/**
+ * This class is of Campaign Page.
+ */
+public class CampaignPage extends BasePage {
+
+    @FindBy(css = ".forceActionLink > div")
+    private WebElement createCampaignBtn;
+
+    @Override
+    protected void waitForPageLoaded() {
+        wait.until(ExpectedConditions.visibilityOf(createCampaignBtn));
```

Comment:
You have to use webelemtaction instead of using a wait directly

Reply:
ok it was solved